cleaners/brave.xml

```diff
@@ -175,4 +174,9 @@
     <action command="sqlite.vacuum" search="file" path="%LocalAppData%\Google\Chrome\User Data\Default\Plugin Data\Google Gears\localserver.db"/>
     <action command="sqlite.vacuum" search="file" path="%LocalAppData%\Google\Chrome\User Data\Default\Plugin Data\Google Gears\permissions.db"/>
   </option>
+  <option id="site_preferences">
+   <label>Site preferences</label>
```

Comment:
It's good that "Site preferences" is already translated

Reply:
I thought it would be good to keep the same name as on Firefox.